### DISCUSSION TITLE
intergrated the EPF and EPD into a chernarus combat ops mission

### DISCUSSION THREAD

**Discussion by kOepi123** [Arkensor/EnfusionPersistenceFramework#39] [General]
Very nice mod you have made there! Thank you for that.
I have a working prototype in basic considerations regarding the persistency.
Now I want to allow the player to spawn wherever he left the game, however it forces him to choose a spawnpoint and the gear he then spawns in is not as he left.

The vanilla setup is that I choose a single loadout from the spawn menu, then select the spawnpoint and then spawn in.
With the integrated and activated EPF setup it does the same thing.

I want the players to spawn where they left off, or at least have the same gear, because weapons are rare.

Best Regards

kOepi

**Comment by Arkensor** [comment]
I am adding integration of combat ops missions and other vanilla content in the base game, so you must not worry about this topic. Just wait for the game to include this.

**Reply by h00** [reply]
has this been implemented yet? If so, can you kindly direct me to any relevant documentation?

**Reply by Arkensor** [reply]
No instead this whole feature is coming to vanilla soon
